### ISSUE TITLE
create new database in distributed system display "java.lang.StackOverflowError"

### ISSUE THREAD

**Issue by youzp** [orientechnologies/orientdb#6942]
## OrientDB Version, operating system, or hardware.
- v2.2.12

## Operating System
- [* ] Linux

## Expected behavior and actual behavior
Five machines consisting of distributed systems, two save vertex data, three save edges data. 
start the distributed system and create a new database display some error
**{"errors":[{"code":505,"reason":505,"content":"java.lang.StackOverflowError"}]}**
## Steps to reproduce the problem
```
{
  "autoDeploy": true,
  "readQuorum": 1,
  "writeQuorum": "majority",
  "executionMode": "asynchronous",
  "readYourWrites": true,
  "servers": {
    "orientdb1":"master",
	"orientdb2":"master",
	"orientdb3":"master",
	"orientdb4":"master",
	"orientdb5":"master"
  },
  "clusters": {
    "internal": {
    },
    "hellovertex": {
      "servers": ["orientdb1","orientdb2"]
    },
	"hellovertex_1": {
      "servers": ["orientdb2","orientdb3"]
    },
	"helloedge": {
      "servers": ["orientdb3","orientdb4"]
    },
	"helloedge_1": {
      "servers": ["orientdb4","orientdb5"]
    },
	"helloedge_2": {
      "servers": ["orientdb5","orientdb1"]
    }
  }
}
```

**Comment by maggiolo00** [comment]
@youzp 

do you have some logs of the server?

**Comment by lvca** [comment]
Could you please use last 2.2.x? We fixed some bugs in the HA part. In case the error persists, could you please send us the logs of the 5 servers in a zip?

**Comment by youzp** [comment]
I modified the configuration, add "*" at the end of the cfg, one new error has occurred.
Three servers are operating normally, but the other two display "**Node is not online yet (status=STARTING), blocking the command until it is online (retry=333, queue=1 worker=7) [ODistributedWorker]**"

[orientdb_log.zip](https://github.com/orientechnologies/orientdb/files/613307/orientdb_log.zip)
The error log 
- orientdb-75.log 
- orientdb-102.log


```
  ip             hostname     orientdb-log-file

- 10.14.133.100  jdorientdb1  orientdb1-100.log
- 10.14.133.103  jdorientdb2  orientdb2-103.log
- 10.14.133.102  jdorientdb3  orientdb3-102.log
- 10.14.133.75   jdorientdb4  orientdb4-75.log
- 10.14.133.104  jdorientdb5  orientdb5-104.log

```



default-distributed-db-config.json
```
{
  "autoDeploy": true,
  "readQuorum": 1,
  "writeQuorum": "majority",
  "executionMode": "asynchronous",
  "readYourWrites": true,
  "servers": {
    "*": "master"
  },
  "clusters": {
    "internal": {
    },
    "hellovertex": {
      "servers": ["jdorientdb1","jdorientdb2"]
    },
	"hellovertex_1": {
      "servers": ["jdorientdb2","jdorientdb3"]
    },
	"helloedge": {
      "servers": ["jdorientdb3","jdorientdb4"]
    },
	"helloedge_1": {
      "servers": ["jdorientdb4","jdorientdb5"]
    },
	"helloedge_2": {
      "servers": ["jdorientdb5","jdorientdb1","<NEW_NODE>"]
    },
	"*": {
      "servers": ["<NEW_NODE>"]
    }
  }
}
```

**Comment by youzp** [comment]
@lvca  @maggiolo00 
Hi, With version 2.2.13, the same cfg and the same error.
In accordance with the order of the first three machines started normal, behind the two machines have been activated to show the error  " **Node is not online yet (status=STARTING), blocking the command until it is online (retry=819, queue=1 worker=3) [ODistributedWorker]**"

Is there a problem with my cfg?

**Comment by youzp** [comment]
hi, @lvca @maggiolo00

I found that someone had mentioned a similar problem. [https://github.com/orientechnologies/orientdb/issues/6666](url)

The past two days I have done a lot of attempts, but there will be the same errors, is this a system bug or my configuration or operational problems?

**Comment by youzp** [comment]
hi, @maggiolo00 @lvca 

I modified the configuration, as shown below, in a cluster with only one server, running the case is normal.
I think it may be a replication problem.
```
{
  "autoDeploy": true,
  "readQuorum": 1,
  "writeQuorum": "majority",
  "executionMode": "undefined",
  "readYourWrites": true,
  "newNodeStrategy": "static",
  "servers": {
    "*": "master"
  },
  "clusters": {
    "internal": {
    },
    "hellovertex_china": {
      "servers": ["jdorientdb1"]
    },
	"hellovertex_usa": {
      "servers": ["jdorientdb2"]
    },
	"helloedge_china": {
      "servers": ["jdorientdb3"]
    },
	"helloedge_usa": {
      "servers": ["jdorientdb4"]
    },
	"helloedge_europe": {
      "servers": ["jdorientdb5"]
    },
	"*": {
      "servers": ["<NEW_NODE>"]
    }
  }
}
```

The logs
|Name        |Status|Databases           |Conns|StartedOn|Binary            |HTTP              |UsedMemory                |
+------------+------+--------------------+-----+---------+------------------+------------------+--------------------------+
|jdorientdb1 |ONLINE|oppo=ONLINE (MASTER)|9    |15:54:28 |10.14.133.100:2424|10.14.133.100:2480|92.07MB/491.00MB (18.75%) |
|jdorientdb2 |ONLINE|oppo=ONLINE (MASTER)|7    |15:23:05 |10.14.133.103:2424|10.14.133.103:2480|102.56MB/471.50MB (21.75%)|
|jdorientdb3 |ONLINE|oppo=ONLINE (MASTER)|9    |15:21:19 |10.14.133.102:2424|10.14.133.102:2480|138.70MB/482.00MB (28.78%)|
|jdorientdb4 |ONLINE|oppo=ONLINE (MASTER)|10   |15:24:31 |10.14.133.75:2424 |10.14.133.75:2480 |142.41MB/463.00MB (30.76%)|
|jdorientdb5*|ONLINE|oppo=ONLINE (MASTER)|7    |15:25:25 |10.14.133.106:2424|10.14.133.106:2480|69.34MB/464.00MB (14.94%) |

**Comment by lvca** [comment]
This has been already fixed in 2.2.14.
